### PR TITLE
highcharts-gantt/1-gantt-axes-groups

### PR DESCRIPTION
**Pull request by Xillez**

### CODE REVIEW

File: highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+                }`;
+            }
+        }
+    },
+    yAxis: {
+        uniqueNames: true
+    },
     series: [{
         name: 'Project 1',
         data: [{
```
Comment (magdalena-gut) [inline highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js:21]: pointWidth seems to be not necessary here

**Review by magdalena-gut — CHANGES_REQUESTED** [review]

File: highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js (hunk 1 above)
Comment (Xillez) [reply highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js:21]: It was in the exercise already but I can remove it for sure :P

Comment (magdalena-gut) [reply highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js:17]: I haven't noticed that! My apologize!

**Review by magdalena-gut — APPROVED** [review]
Perfect! Good job!

File: highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js (hunk 1 above)
Comment (Xillez) [reply highcharts-api/highcharts-gantt/1-gantt-axes-groups/main.js:17]: No worries

**Pull request merged**